paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/AddPaymentMethod.kt

```diff
@@ -107,6 +108,9 @@ internal fun AddPaymentMethod(
                     linkSignupState = inlineSignupViewState
                 },
                 formArguments = arguments,
+                usBankAccountFormArguments = USBankAccountFormArguments(
+                    onMandateTextChanged = sheetViewModel::updateBelowButtonText
```

Comment:
Can we move `updateBelowButtonText`, and it's associated field into `USBankAccountFormArguments` as well?

Reply:
I don't think that the USBankAccountForm should care about updateBelowButtonText